.github/workflows/CI.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  build_zip:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ghidra: [ 'latest', '10.1.3', '10.1.2', '10.1.1' ]
+    steps:
+      - name: Set Ghidra version
+        id: ghidra_version
+        run: |
+          if [ ${{ matrix.ghidra }} = latest ]; then
+            ghidra_version=$(curl -fLsSo /dev/null -w %{url_effective} https://github.com/NationalSecurityAgency/ghidra/releases/latest |
+             grep -oP '(?<=https://github.com/NationalSecurityAgency/ghidra/releases/tag/Ghidra_).+(?=_build)')
+            echo "::set-output name=full::$ghidra_version"
+          else
+            echo "::set-output name=full::${{ matrix.ghidra }}"
+          fi
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Checkout pspdev/psp-ghidra-scripts
+        uses: actions/checkout@v3
+        with:
+          repository: 'pspdev/psp-ghidra-scripts'
+          path: 'psp-ghidra-scripts'
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: temurin
+      - name: Setup Ghidra
+        uses: er28-0652/setup-ghidra@master
```

Comment:
Let's pin it to a specific version instead of using `master`. I guess this just exports `GHIDRA_INSTALL_DIR`? (maybe we can get away without it altogether)

Reply:
Yeah

Reply:
> Let's pin it to a specific version instead of using master.

Comment still stands.